community/cloud-foundation/templates/gcs_bucket/tests/integration/gcs_bucket.bats

```diff
@@ -64,7 +64,7 @@ function teardown() {
         gcloud --quiet iam service-accounts delete "${SA_NAME}"
     fi
 
-    # Per-test teardown steps here
+    # Per-test teardown steps.
 }
 
 ########## TESTS ##########
```

Comment:
Looks like something is wrong with the test YAML:

```
 ✗ Creating deployment ocsi-courseralab1-guestbook-gcs-bucket-y32jhs3bj1 from .ocsi-courseralab1-guestbook-gcs-bucket-y32jhs3bj1.yaml
   (in test file templates/gcs_bucket/tests/integration/gcs_bucket.bats, line 73)
     `gcloud deployment-manager deployments create "${DEPLOYMENT_NAME}" \' failed
   Creating .ocsi-courseralab1-guestbook-gcs-bucket-y32jhs3bj1.yaml
   Created service account [test-bucket-y32jhs3bj1].
   ERROR: (gcloud.deployment-manager.deployments.create) Failed to parse YAML: while scanning a simple key
     in "<unicode string>", line 164, column 19:
                         or older.
                         ^ (line: 164)
   could not find expected ':'
     in "<unicode string>", line 165, column 17:
                       createdBefore:
                       ^ (line: 165)
```

It failed to create so all tests are failing atm.

Reply:
Hi @fawix, seems like somehow the `gcs_bucket.py.schema` file somehow got creeped with `???` chars into them. As a result, the schema's format was thrown off and hence the error

However, neither this branch `gcs_bucket_doc` nor the `cloud-foundation` branch doesn't seem to have those chars .. at least that's what I see now. This could also be due to issues with Github at the moment, that I cannot see those chars on this branch.

Can you verify the `gcs_bucket.py.schema` file and search for any chars that resemble `???` or for any funny chars in the lines (164, 165) - if you find them can you remove them and try them again please? I just ran into the same issue locally (but mine were on lines 51..55) I removed those chars, ensure the schema is a valid yaml and then re-ran the BATS test and they pass

![image](https://user-images.githubusercontent.com/42005789/47309234-e4fe9100-d601-11e8-8d8a-18d05555088a.png)

Reply:
I searched all over I can't see any `???` (I'm using vscode); can you try committing your file see if I pick up the changes? thx

Reply:
it worked!